climada_petals/hazard/wildfire.py

```diff
@@ -383,22 +403,29 @@ def set_proba_fire_seasons(self, n_fire_seasons=1, n_ignitions=None,
             ign_max = n_ignitions[1]
 
         prob_fire_seasons = [] # list to save probabilistic fire seasons
+        event_proba_new = [] #added by Sam. G.
         # create probabilistic fire seasons
         for i in range(n_fire_seasons):
+            self.ProbaParams.prop_proba = float(np.random.normal(self.ProbaParams.prop_proba_mean,
+                                                           self.ProbaParams.prop_proba_std,1)) # added by Sam G.
             n_ign = np.random.randint(ign_min, ign_max)
-            LOGGER.info('Setting up probabilistic fire season with %s fires.',\
-                        str(n_ign))
+            LOGGER.info('Setting up probabilistic fire season with %s fires. Global proba = %.3f'
+                        %(n_ign,self.ProbaParams.prop_proba))  #changed by Sam. G.
+            event_proba_new.append(self.ProbaParams.prop_proba) #added by Sam. G.
+            LOGGER.info('Fire season: %i' %(i+1)) # added by Sam G.
```

Comment:
This looks more like a debugging message than info. Is this really needed? Just a thought..